python/ray/workflow/http_event_provider.py

```diff
@@ -213,7 +213,8 @@ async def event_checkpointed(self, event) -> None:
     Example Usage
     =============
 
-    .. code-block:: python
+    .. testcode::
+        :skipif: True
```

Comment:
Would it be easy to add an implementation for `handle_event`? If so, do you think we could unskip this?